docs/store/api.md

```diff
@@ -158,3 +158,27 @@ export const reducers: ActionReducerMap<fromFeature.State> = {
 })
 export class FeatureModule { }
 ```
+
```

Comment:
Add a "Injecting Meta-Reducers" header for this section and add a link in the TOC at the bottom of the README.md please.